ext/tags_test.go

```diff
@@ -1,20 +1,27 @@
 package ext_test
 
 import (
+	"reflect"
 	"testing"
 
+	"github.com/opentracing/basictracer-go"
+	"github.com/opentracing/basictracer-go/testutils"
```

Comment:
why not just use noop tracer? I think the only point of these tests (at least in python) was to get test coverage to 100%. In Go they have additional benefit of validating signatures, but we don't need asserts to read back the values imo.